salt/docker/init.sls

```diff
@@ -23,10 +23,10 @@ include:
 #######################
 
 {% set docker_opts = salt['pillar.get']('docker:args', '')%}
-{% set docker_reg  = salt['pillar.get']('docker:registry', '') %}
-{% if docker_reg|length > 0 %}
-  {% set docker_opts = docker_opts + " --insecure-registry={{ docker_reg }} --registry-mirror=http://{{ docker_reg }}" %}
-{% endif %}
+{% set docker_regs  = salt['pillar.get']('docker:registry', '').split(',') %}
+{% for reg in docker_regs if docker_regs|length > 0 %}
+  {% set docker_opts = docker_opts + " --insecure-registry={{ reg }} --registry-mirror=http://{{ reg }}" %}
```

Comment:
tried to look back in [history](https://github.com/kubic-project/salt/commit/4d8f25bdafe0e6ba044548fa44b2098f60c1bb5b) but the commit message wasn't descriptive enough. maybe @inercia can provide some details.

would it make sense to separate `--insecure-registry` and `--registry-mirror` in 2 pillars? seems there could be a use case where you may want one **or** the other